source/develop/release-management/features/network/provider-physical-network.md

```diff
@@ -79,6 +79,20 @@ Engine also covers the validation:
 - Physical network must be on the same Data Center as the external network.
 - The custom values (physical network and VLAN ID) must not be specified a when provider physical network is set.
 
+To import a network with physical network access from an external network
```

Comment:
Maybe we can add a subsection for import and synchronization (how it behaves on engine with physical networks)

Reply:
Done.